frontend/public/components/utils/resource-log.jsx

```diff
@@ -111,7 +116,14 @@ export const LogControls = ({
               </React.Fragment>
             );
           })}
-        <Button variant="link" isInline onClick={onDownload}>
+        <Button component="a" variant="link" isInline href={logURL} target="_blank">
+          <OutlinedWindowRestoreIcon className="co-icon-space-r" />
+          Raw
+        </Button>
+        <span aria-hidden="true" className="co-action-divider hidden-xs">
+          |
+        </span>
+        <Button component="a" variant="link" isInline href={logURL} download>
```

Comment:
No reason to use `component="a"` here. Semantically it's a button.

Reply:
Changed to a link since href/download props don't work on button.